osh.spec

```diff
@@ -222,13 +231,17 @@ ln -s %{_sysconfdir}/osh/hub/settings_local.py %{buildroot}%{python3_sitelib}/os
 %{bash_completions_dir}
 %{zsh_completions_dir}
 %{python3_sitelib}/osh/client
-%{python3_sitelib}/osh-*-py%{python3_version}.egg-info
 
 %files common
 %dir %{_sysconfdir}/osh
 %{python3_sitelib}/osh/common
 %{python3_sitelib}/osh/__init__.py*
 %{python3_sitelib}/osh/__pycache__
+%if 0%{?fedora} > 41
+%{python3_sitelib}/osh-*.dist-info
+%else
+%{python3_sitelib}/osh-*-py%{python3_version}.egg-info
```

Comment:
Why was this also moved to a different subpackage?

Reply:
It might have been wrong previously (and nobody cared).  To be honest, I have no idea why we need to package these files.  My impression was that they should be installed whenever the related Python modules are installed, which means `osh-common` in this case.

Are you proposing to move them back to `osh-client`?  Would that improve anything?